R/guide-legend.r

```diff
@@ -694,7 +691,8 @@ guide_gengrob.legend <- function(guide, theme) {
   )
   gt <- gtable_add_grob(
     gt,
-    justify_grobs(grob.title, hjust = title.hjust, vjust = title.vjust, debug = title.theme$debug),
+    justify_grobs(grob.title, hjust = title.hjust, vjust = title.vjust,
```

Comment:
Can you please use modern tidyverse style here?

Reply:
Will do.